packages/connect/src/data/defaultFeeLevels.ts

```diff
@@ -0,0 +1,130 @@
+import { FeeLevel, FeeInfo } from '../types';
+
+// this is workaround for the lack of information from 'trezor-common'
+// we need to declare what does "high/normal/low" mean in block time (eg: normal BTC = 6 blocks = ~1 hour)
+// coins other than BTC usually got 2 levels maximum (high/low) and we should consider to remove other levels from 'trezor-common'
+const BLOCKS_FOR_FEE_LEVEL: Record<string, Record<string, number>> = {
+    btc: {
+        // blocktime ~ 600sec.
+        high: 1,
+        normal: 6,
+        economy: 36,
+        low: 144,
+    },
+    bch: {
+        // blocktime ~ 600sec.
+        high: 1,
+        normal: 5,
+        economy: 10,
+        low: 10,
+    },
+    btg: {
+        // blocktime ~ 600sec.
+        high: 1,
+        normal: 5,
+        economy: 10,
+        low: 10,
+    },
+    dgb: {
+        // blocktime ~ 20sec.
+        high: 1,
+        normal: 15,
+        economy: 30,
+        low: 60,
+    },
+};
+
+const getDefaultBlocksForFeeLevel = (shortcut: string, label: string) =>
+    BLOCKS_FOR_FEE_LEVEL[shortcut] && BLOCKS_FOR_FEE_LEVEL[shortcut][label]
+        ? BLOCKS_FOR_FEE_LEVEL[shortcut][label]
+        : -1; // -1 for unknown
+
+// partial data from coins.jon
+interface CoinsJsonData {
+    shortcut: string; // uppercase shortcut
+    // data below are defined and relevant only for bitcoin-like coins
+    blocktime_seconds: number;
+    default_fee_b: Record<'High' | 'Normal' | 'Economy' | 'Low', number>;
+    maxfee_kb: number;
+    minfee_kb: number;
+    dust_limit: number;
+}
+
+export type FeeInfoWithLevels = FeeInfo & { defaultFees: FeeLevel[] };
+
+export const getBitcoinFeeLevels = (coin: CoinsJsonData): FeeInfoWithLevels => {
+    // sort fee levels from coinInfo
+    // and transform in to FeeLevel object
+    const defaultFees = coin.default_fee_b;
+    const shortcut = coin.shortcut.toLowerCase();
+    const keys = Object.keys(defaultFees) as unknown as (keyof CoinsJsonData['default_fee_b'])[];
+    const levels = keys
+        .sort((levelA, levelB) => defaultFees[levelB] - defaultFees[levelA])
+        .map(level => {
+            const label = level.toLowerCase() as FeeLevel['label']; // string !== 'high' | 'normal'....
+            return {
+                label,
+                feePerUnit: defaultFees[level].toString(),
+                blocks: getDefaultBlocksForFeeLevel(shortcut, label),
+            };
+        });
+
+    return {
+        blockTime: coin.blocktime_seconds / 60,
```

Comment:
blocktime is blocktime in minutes?

I undrestand that doing Math.round (which was tehre before) was at least inaccurate since some very fast chains might have been rounded down to zero.

But wouldn't it make sense to round it to certain number of decimals here? probably not needed I guess. Just that I spotted a difference here.

Reply:
again right, this value is used by suite send form:
```
formatDuration(feeInfo.blockTime * selectedLevel.blocks * 60)
```
it will be safer to return this value in minutes (int) - so now im rounding to 0 (as it was) but picking up Math.max between 1 and real value [ffc3f85](https://github.com/trezor/trezor-suite/pull/9171/commits/ffc3f85e559865e7801fa81151a0cf4c4086de42) + [6e533ef](https://github.com/trezor/trezor-suite/pull/9171/commits/6e533ef66894f0ae9e9acfa08e9368cc151d02bb)

Reply:
i think two last fixups will throw conflicts while squashing, so i need to suqash it manually and force push it